libr/asm/arch/arm/armass.c

```diff
@@ -330,6 +330,60 @@ static int getlist(char *op) {
 	return list;
 }
 
+static ut32 thumb_getshift(const char *str) {
+	// only immediate shifts are ever used by thumb-2. Bit positions are different from ARM.
+	const char *shifts[] = {
+		"LSL", "LSR", "ASR", "ROR", 0, "RRX"
+	};
+	char *type;
```

Comment:
dont define variables at the begining without value, we use c99 so you can declare and assign in the same place.

for example: char *type = strdup (str);